packages/gatsby/.eslintrc.js

```diff
@@ -6,5 +6,6 @@ module.exports = {
   parserOptions: {
     jsx: true,
   },
+  ignorePatterns: ['scripts/prepack.ts'],
```

Comment:
I admit, I sort of tried to take the easy way out here because eslint would throw an error that the Gatsby-specific `prepack.ts` file didn't have a `tsconfig` which it apparently needs 🤔 
I'll dig deeper to see what we actually have to do. So far I tried to point it to our Gatsby tsconfig (where I also needed to include `prepack.ts` in) but afaik this would also compile the file which we don't want either. Anyway, if you have an easy workaround for this, let me know. I'll do some more research in the meantime

Reply:
Yup exactly